api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupSMCRegister.java

```diff
@@ -128,27 +141,51 @@ public String getVersion() {
         return l;
     }
 
+    private CohortDefinition getSmcUsedEncounters(){
+        CompositionCohortDefinition cd = new CompositionCohortDefinition();
+        cd.addParameter(new Parameter("startDate", "Start Date", Date.class));
+        cd.addParameter(new Parameter("endDate", "End Date", Date.class));
+        cd.addSearch("client", ReportUtils.map(Cohorts.genderAndHasAncEncounter(false, true, "244da86d-f80e-48fe-aba9-067f241905ee"), "startDate=${startDate},endDate=${endDate}"));
```

Comment:
I reused this method instead of duplicating code. I will refactor and rename it to be general after merging because it is used somewhere else in ANC register but those encouter types used are for SMC